tox.ini

```
@@ -10,8 +10,11 @@ basepython = python3
 setenv =
     PYTHONPATH={toxinidir}:{toxinidir}/lib
 deps =
+    jinja2
+    netaddr<=0.7.19
     pyyaml
     pytest
+    pytest-mock
     pytest-cov
     flake8
     git+https://github.com/juju-solutions/charms.unit_test/#egg=charms.unit_test
```

This has been released to PyPI so could be updated to:

```suggestion
    charms.unit_test
```